app/assets/javascripts/dash_main.js

```diff
@@ -33,3 +33,4 @@
     }
   }
 })
+;
```

Comment:
Sorry, bad habit. I learned JS circa 2009 so it's largely just muscle memory :stuck_out_tongue: